apps/api/.env.local.example

```
@@ -20,4 +20,7 @@ PORT=
 GOOGLE_GENERATIVE_AI_API_KEY=
 
 # Redis
-REDIS_URL=redis://localhost:6379
\ No newline at end of file
+REDIS_URL=redis://localhost:6379
+
+# Reflag (Feature Flags)
+REFLAG_SECRET_KEY=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)